src/disco/containers/DiscoPane.js

```diff
@@ -20,6 +20,21 @@ class DiscoPane extends React.Component {
     const { results } = this.props;
     return (
       <div id="app-view" ref="container">
+        <header>
+          <div className="content">
+            <h1>{_('Personalize Your Firefox')}</h1>
+            <p>{_(`There are thousands of add-ons that let you make Firefox all your
```

Comment:
These new lines seem slightly concerning of translating. changing the indent could result in the string changing but maybe that isn't worth worrying about until we pick an extraction library.